index.html

```diff
@@ -712,23 +712,23 @@ <h2 class='informative'>Vary</h2>
       <h2>Resource Authorization</h2>
       <p>
         To configure access control restrictions, implementations MUST follow the recommendations
-        of [[!SOLIDWEBAC]].
+        of [[!SOLIDWEBAC]] with the following additional requirements:
       </p>
-      <p>
-        Implementations supporting access control MUST use the [[!WEBAC]] model to
-        configure that functionality. Any access control resource in use MUST be a <a>LDP-RS</a>,
-        which SHOULD be located in the same server as the controlled resource. Implementations
-        MUST support extended control relationships in which the access control resource for some
-        resource has its own access control resource and so on. Implementations MUST make access
```

Comment:
Yes, but please change the formatting. A "click here" link is really not best practice.